### PR TITLE
feat: add scraping daemon to periodically run scripts

### LINKED ISSUE #15: Create a single script to run all the scripts periodically
Create a single script (shell or python) to run scripts for all the cities periodically.
The script has to execute the files for each region (currently `bangalore.py`, `pune.py` and `andhra.py`) periodically (every hour).

*Note: It should be an infinite loop*

### PR DESCRIPTION
**Pull request by sudiptog81**
* added a daemon to run the scripts for the cities every hour
* updated the production Dockerfile to reflect the changes

closes #15.

### CODE REVIEW

File: scripts/scrape.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+cities = [
+    "bangalore",
+    "pune",
+    "andhra"
+]
+
+while True:
+    for cmd in [f"python {city}.py" for city in cities]:
+        os.system(cmd)
```
Comment (shreyas44) [inline scripts/scrape.py:12]: This looks good! Could you just add a line to log 'waiting for another hour to update...' or something similar?

**Comment by shreyas44** [comment]
@sudiptog81 This looks good, thank you!

**Review by shreyas44 — APPROVED** [review]

**Comment by vivekshah1801** [comment]
@shreyas44 
Though the issue is closed, I want to suggest a cronjob for this.

Some points that you can consider for cronjob over infinite loop.
- [x] scrape.py need not to sit in memory for the idle time.
- [x] sepration of concerns

**Comment by shreyas44** [comment]
@vivekshah1801 thank you for the suggestion!

I can see how using cronjob would reduce memory usage, not by much but it would. I don't see how using cronjob would create a further separation of concerns, could you further comment on that?

You're welcome to raise an issue regarding this and submit a PR with a shell script running the cronjob 🙂

**Comment by vivekshah1801** [comment]
@shreyas44 
We are basically creating loop for scheduling the script-run at particular intevals.

Cronjob will handle the scheduling (runniing the script at periodic intervals)
while main python script will coordinate the logic flow (running scapping job)

This is how we can seprate scheduling and business logic.

**Comment by shreyas44** [comment]
Ahh okay got it. Will you be submitting a PR implementing the same?

**Comment by vivekshah1801** [comment]
Yes.

**Comment by shreyas44** [comment]
@all-contributors add @sudiptog81 for code

**Pull request merged**